catalog/spatial/kml/spatial-kml-util/src/main/java/org/codice/ddf/spatial/kml/converter/MetacardToKml.java

```diff
@@ -111,16 +115,22 @@ private static Geometry createPointGeo(Point jtsPoint) {
     return KmlFactory.createPoint().addToCoordinates(jtsPoint.getX(), jtsPoint.getY());
   }
 
-  private static com.vividsolutions.jts.geom.Geometry readGeoFromWkt(final String wkt)
+  public static com.vividsolutions.jts.geom.Geometry getJtsGeoFromWkt(final String wkt)
```

Comment:
✏️ if `wkt` can be `null` then it should be annotated with `@Nullable`. If not then we can replace the `StringUtils.isBlank(wkt)` below with `wkt.isEmpty()`